backends/p4tools/modules/testgen/targets/bmv2/backend/ptf/ptf.cpp

```diff
@@ -404,10 +404,10 @@ class Test{{test_id}}(AbstractTest):
 ## else 
         ptfutils.verify_packet(self, exp_pkt, eg_port)
         bt.testutils.log.info("Verifying no other packets ...")
-        ptfutils.verify_no_other_packets(self, self.device_id, timeout=2)
+        ptfutils.verify_no_other_packets(self, self.device_id, timeout=0.1)
```

Comment:
I would recommend maybe using a global variable name or a return value from a global function defined with `def` at the top level to get this timeout value, so that after a PTF test is generated, a user can replace this number in one place in the PTF source file, rather than many places.

Similarly if there are other timeouts to wait for a different reason.

Reply:
Rationale: 0.1 sec might work right now for BMv2, but I've been testing with other environments where it is definitely not long enough.

Reply:
What are the environments where issues are coming up?